en/complete_vehicles/atl_mantis_edu.md

```diff
@@ -0,0 +1,139 @@
+# ATL Mantis EDU
+
+:::warning
+The ATL Mantis EDU is currently work in progress. Use with caution.
+:::
+
+The ATL Mantis EDU is based on the ready-to-fly Yuneec Mantis G and geared towards education.
+
+## Purchase
+
+The ATL Mantis EDU version will be released soon.
+TODO: add purchase link
+
+## Hardware overview
+
+The Mantis broadly features the following hardware components:
+- Autopilot/FMU based on STM32F7.
```

Comment:
So not a separately "for sale" FC that we can link to?

Reply:
Nope.